.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 python:
     - 3.5
     - 3.6
+    - 3.7
```

Comment:
Please also add it to `allowed_failures:` until travis-ci/travis-ci#9815 resolved. And you may also upvote that request :)

Reply:
Ok, will do!

Reply:
Never really done Travis before, is it
```py
matrix:
  allow_failures:
  - python: 3.7
```

Reply:
Nevermind, I'll fix it myself. I've found a way to actually run it in Travis.

Reply:
but the answer to your question would be yes, that's a correct way to do it.